### DISCUSSION TITLE
Cant compile

### DISCUSSION THREAD

**Discussion by Borming** [rainbow-me/rainbowkit#964] [Q&A]
I have a simple react 17 App with simple import 

import { WagmiConfig, createClient, configureChains, mainnet } from 'wagmi';

I am getting this error when compiling.

ERROR in ./node_modules/wagmi/dist/index.js 188:33
Module parse failed: Unexpected token (188:33)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
|   if (defaultedOptions.suspense && result.isLoading && result.isFetching && !isRestoring) {
|     throw observer.fetchOptimistic(defaultedOptions).then(({ data }) => {
>       defaultedOptions.onSuccess?.(data);
|       defaultedOptions.onSettled?.(data, null);
|     }).catch((error) => {
ℹ ｢wdm｣: Failed to compile.

Anyone knows whats going on?

**Comment by praddoskanna** [comment]
Facing Same Issue .
